plugins/communications-app/ScheduleSection/index.jsx

```diff
@@ -23,8 +24,9 @@ import { useSelector, useDispatch } from '@communications-app/src/components/bul
 import { actionCreators as formActions } from '@communications-app/src/components/bulk-email-tool/bulk-email-form/BuildEmailFormExtensible/context/reducer';
 
 import messages from './messages';
+import './styles.scss';
 
-const formStatusToast = ['error', 'complete', 'completeSchedule'];
+const formStatusToast = ['error', 'complete', 'completeSchedule', 'loadingTeams'];
```

Comment:
I've seen this status in many different places, I would suggest to move into a variables, in order to avoid typos in the future.

Ej:

```suggestion
const ERROR = 'error'
const COMPLETE = 'complete'
const  COMPLETE_SCHEDULE = 'completeSchedule'
const LOADING_TEAMS = 'loadingTeams'
const formStatusToast = [ERROR, COMPLETE, COMPLETE_SCHEDULE, LOADING_TEAMS];
```